### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by FrontendPatriciaAtRolo**
Update Close/MicOnBorderless/MicOffBorderless icons by passing props.fill
Rename Icons as pic
image
image

### CODE REVIEW

**Comment by lynnchi94** [comment]
Will all the icons have some customized color or other properties?
If so, I think we can pass the same props options to all the icons.
@Jacky-Rolo @FrontendPatriciaAtRolo what do you think?

File: src/components/Icons/MicOff.js [hunk 1]
```diff
@@ -21,3 +22,15 @@ export function Mute(props) {
     </Svg>
   );
 }
+
+MicOff.propTypes = {
+  size: PropTypes.number,
+  color: PropTypes.string,
+  hasBorder: PropTypes.bool,
```
Comment (Jacky-Rolo) [inline src/components/Icons/MicOff.js:29]: remove useless propTypes `size`, `hasBorder`

**Comment by Jacky-Rolo** [comment]
> Will all the icons have some customized color or other properties?

for now, **ONLY** few of them

> If so, I think we can pass the same props options to all the icons.

I want, but seems **NO** ONE logic can apply **ALL** Icons.  
looks like "pass what props to what element" have to judgment by developer.   
which means we have discuss all icons **ONE BY ONE**  

I think It's NOT our TOP priority.

File: src/components/Icons/MicOff.js (hunk 1 above)
Comment (FrontendPatriciaAtRolo) [reply src/components/Icons/MicOff.js]: b9caa7cfb14b4d8bf2ddabacd75ca8ddefede3f1

**Review by lynnchi94 — APPROVED** [review]

**Pull request merged**